tests/test_bitmaps.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+#coding:utf-8
+# Author:  Ross Golder --<ross@golder.org>
+# Purpose: test_bitmaps
+# Created: 23/04/2014
+# Copyright (C) 2014, Ross Golder
+# License: BSD licence

```

Comment:
can't land it with this stuff I'm afraid, please don't put licensing stuff in your commits, if you're not happy to give over your code, you're not allowed to play, I'm afraid...